db/controllers/articles.controllers.js

```diff
@@ -41,9 +41,29 @@ const addCommentToArticle = (req, res, next) => {
         });
 };
 
+const patchArticleVotes = (req, res, next) => {
+    const { article_id } = req.params;
+    const { inc_votes } = req.body;
+
+    if (typeof inc_votes !== 'number') {
+        return res.status(400).send({ msg: 'Invalid input' });
+    }
```

Comment:
If you get some error handling middleware and look at the error code that PSQL sends back, you shouldn't need to have this conditional logic here. We also shouldn't be handling errors in our controllers anyway.